billboard.py

```diff
@@ -65,7 +63,7 @@ def __repr__(self):
             s = u"%s" % self.artist
 
         if sys.version_info.major < 3:
-            return s.encode(getattr(sys.stdout, 'encoding', '') or 'utf8')
+            return s.encode('utf8' or getattr(sys.stdout, 'encoding', ''))
```

Comment:
What's the reasoning behind this?

Reply:
Somewhere along my testing I was running into the situation where `getattr(sys.stdout, 'encoding', '')` was returning 'US-ASCII', which gave me UnicodeEncodeError (for cases like 'Volaré'), so I just switched the order as a quick fix. I didn't go back to check this on my final commit. Checking it now, `getattr(sys.stdout, 'encoding', '')` now returns 'UTF-8'. If needed, the order can actually be reverted to original.

Reply:
Okay, reverted in a1209cde6d0281c191f916e9bad7a73c0e864e7d.